react/components/organisms/chat/chat.js

```diff
@@ -209,10 +212,7 @@ export class Chat extends PureComponent {
     _renderNoMessages = () => {
         return (
             <View style={styles.noMessages}>
-                <Image
-                    style={styles.noMessagesImage}
-                    source={require("./assets/no-messages.png")}
-                />
+                <WebView style={styles.noMessagesImage} source={noMessageAnimation} />

```

Comment:
add missing prop here to fix navigation issues
and also... isn't something missing for the `source` prop?
```suggestion
                <WebView style={styles.noMessagesImage} source={{ html: noMessageAnimation }} androidLayerType={"software"} />

```